test/integration/controller_test.go

```diff
@@ -739,6 +813,10 @@ type controllerTest struct {
 	// reconciliation is expected to fail.
 	skipVerifyingBindingSuccess bool
 
+	skipVerifyingBrokerDeleteSuccess   bool
```

Comment:
I would argue pretty strongly against these being in here, **ever** let alone now when they are still not being used. If the tear down is going to skip verification of the delete, then that implies that the delete is going to fail. In that case, the tear down may as well not do the delete at all. That can be accomplished by clearing out the appropriate field in the `testController`.

For example, let's say that the test is left in a state where we expect the delete of the `ServiceInstance` to fail. In that case, there is no reason for the test to even attempt to delete the `ServiceInstance`. Instead of setting `skipVerifyInstanceDeleteSuccess` to true, just set `instance` to nil, thereby skipping the delete of the `ServiceInstance`.

Moreover, I would argue that if there is a test that is being left in a state where we expect the delete of the resource to fail, that test *should* be attempting a delete and verifying that the delete did indeed fail.